scripts/apps/authoring/multiversion/directives/MultiversionFloatMenuDirective.js

```diff
@@ -0,0 +1,92 @@
+class LinkFunction {
+    constructor($document, scope, elem) {
+        this.$document = $document;
+        this.scope = scope;
+        this.elem = elem;
+
+        this.init();
+    }
+
+    /**
+     * @ngdoc method
+     * @name sdMultiversionFloatMenu#init
+     * @private
+     * @description Initializes the directive with default values for the scope
+     * and with necessary watchers.
+     */
+    init() {
+        this.open = false;
+
+        this.elem.bind('click', (event) => {
+            if (!this.open) {
```

Comment:
I would put this event listener into a function called `toggle`. Or, alternatively, I would use two methods `show` and `hide`. `hide` could replace `closeOnClick` and line 30 would not be needed anymore because show and hide would set the value of `this.open` accordingly.